src/io/flutter/FlutterUtils.java

```diff
@@ -440,4 +444,84 @@ protected void addImplicitResolvers() {
       return null;
     }
   }
+
+  public static boolean isAndroidxProject(@NotNull Project project) {
+    @SystemIndependent String basePath = project.getBasePath();
+    assert basePath != null;
+    VirtualFile projectDir = LocalFileSystem.getInstance().findFileByPath(basePath);
+    assert projectDir != null;
+    VirtualFile androidDir = getFlutterManagedAndroidDir(projectDir);
+    if (androidDir == null) {
+      return false;
+    }
+    VirtualFile properties = androidDir.findChild("gradle.properties");
```

Comment:
Based on a search of the intellij-community code it looks like you can use
import org.gradle.wrapper.SystemPropertiesHandler;
to parse the gradle.properites handler instead of having to parse it yourself here.

Reply:
Good point.